### PR TITLE
Update BUILDING.md

### PR DESCRIPTION
**Pull request by phydroxide**
Package list for Yum incomplete

### CODE REVIEW

File: BUILDING.md [hunk 1]
```diff
@@ -86,6 +86,10 @@ $ sudo make install
 ```
 
 This should complete without errors, and `sqlitebrowser` should now be launch-able from the command line.
+**Note 3**
```
Comment (MKleusberg) [inline BUILDING.md]: Can you add an extra empty line above the ```Note 3``` line? Otherwise the layout is slightly broken :wink:

**Comment by MKleusberg** [comment]
Awesome! Thanks again for this, @phydroxide 😄

**Comment by justinclift** [comment]
There are some mistakes in the package names. 😦 

@phydroxide Which version of CentOS is this for?  I'll install it in a VM and see how it goes there with and without this change. :smile:

**Comment by phydroxide** [comment]
Cent OS 7 x64
I used the 1708 everything iso 
Minimal install plus server GUI packages

**Comment by justinclift** [comment]
Thanks @phydroxide.  I'll give that a go in a VM now. :smile:

**Comment by justinclift** [comment]
Ok, on a brand new CentOS 7 VM, running the new `sudo yum ...` command immediately gives these warnings:

```
No package qt-dev available.
No package qwt-qt5-devel available.
No package sqllite-devel available.
No package qt5widgets available.
```

The existing instructions give this warning:

```
No package qwt-qt5-devel available.
```

So, at the very least the CentOS steps should be adjusted to not include `qwt-qt5-devel` for CentOS 7.

When going through the steps using the instructions prior to this PR... for me in a newly installed CentOS 7 x64 Minimal install + server GUI packages, DB4S compiled and runs fine.

Completely guessing here, but my suspicion is you forgot to add "qt5-qtbase-devel` (from **Note 2**) to the yum line?  And maybe missed the `sqlite-devel` package name too (on the 2nd line of the cut-n-paste steps)?

**Comment by justinclift** [comment]
@phydroxide I'm going to remove the new **Note 3**, as it's not needed, but will also tweak the existing **Note 2** to workaround the `qwt-qt5-devel` warning.  Thanks for submitting the PR though.  It was a good attempt. :smile:

**Pull request merged**